scripts/cmd-all

```diff
@@ -23,69 +23,11 @@ echo "------------------------------------------------------------------------"
 
 cargo $sub_cmd -p spacewalk-primitives --all-features $options
 
-echo "------------------------------------------------------------------------"
```

Comment:
Broken down this script into multiple ones, to avoid the issue aforementioned in the description.